requirements.txt

```diff
@@ -1,7 +1,8 @@
 beautifulsoup4~=4.7
 cld2_cffi~=0.1
 datasketch==1.4.0
-gensim==3.7.2
+gensim==3.8.0
+numpy==1.16.4
```

Comment:
is there a reason we don't go for `~=` for `gensim`, `numpy` (and `datasketch` but ok)?
And how is it possible `numpy` wasn't in reqs, but used? (was it installed as a dependency? 😬)

Reply:
Why would we go with an approximate version if we have an absolute version that works?

Yes, `numpy` was installed as a dependency, but as we're using it directly it should be in reqs as well.

Reply:
Approximate because updates may fix things we do not know are broken?

And that `numpy` thing oh gosh, it's one of the few libs that can yield drama when versions change, and we had no control at all over it 😬! So good.

Reply:
Good point, changed versioning to approximate versioning!